src/NLog/Common/LogEventInfoBuffer.cs

```diff
@@ -29,7 +29,6 @@
 // CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF 
 // THE POSSIBILITY OF SUCH DAMAGE.
-// 
```

Comment:
Little curious why this was removed? This this PR should be without modifications of this file.

Reply:
My mistake probably.
On it.